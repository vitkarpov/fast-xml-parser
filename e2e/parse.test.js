const fs = require('fs');
const {parse} = require('../dst/index');

describe('parse', () => {
  fs.readdirSync('./e2e/inputs/').map((file) => {
    return require(`./inputs/${file}`)
  }).forEach((input) => {
    it(input, () => {
      const root = parse(input);

      expect(root).toMatchSnapshot();
    });
  });
});
