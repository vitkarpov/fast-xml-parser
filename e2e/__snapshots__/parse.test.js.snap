// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`parse 
  <html>
    <head>
      <title>Hello, world!</title>
    </head>
    <body>
      <h1>Homepage</h1>
    </body>
  </html>
 1`] = `Node {}`;
